### DISCUSSION TITLE
Plain text editor

### DISCUSSION THREAD

**Discussion by amiranagram** [ueberdosis/tiptap#845] [General Questions] [answered]
Hi, is there any option I could use tiptap as a plain text input? I really like how you can customize the editor with the extensions and I have made a minimal design that I like a lot, but I wanna get the content as a plain text and also disable new paragraph on return, etc.

Where it says Add Title, that is the editor instance I want to use as plain text.

![image](https://user-images.githubusercontent.com/38536188/94448501-41ce3300-01ab-11eb-9bce-75e232438c2e.png)

**Comment by hanspagel** [comment, marked as answer]
Oh, I’m not sure if that is possible. A few things you could do:

* Remove the padding from paragraphs, so they all look like a new line.
* Replace the paragraph extension with something custom, but it’ll need to render a HTML tag anyway.
* Remove the HardBreak extensions, so you have no `<br>` anymore.
* Store the HTML, but remove all tags when you render them.

That‘s all I have in mind. If you’d like to have a Markdown editor, have a look at CodeMirror.
